module #2. REST API Basics/rest-api-basic/core/src/main/java/com/epam/esm/exception/model/GiftCertificateAlreadyExistsException.java

```diff
@@ -0,0 +1,18 @@
+package com.epam.esm.exception.model;
+
+import com.epam.esm.exception.AbstractException;
+
+public class GiftCertificateAlreadyExistsException extends AbstractException {
+    private static final long serialVersionUID = -5057233134287184159L;
```

Comment:
There is no need to specify serialVersionUID because those classes will not be serialized. This one is redundant